### PR TITLE
31113 popover nightly

### PR DESCRIPTION
**Pull request by dletorey**
### Description

Added relavant see also links to the various pages that relate to popover

### Motivation

Working on making sure popover is documented for [default release in nightly](https://github.com/mdn/content/issues/31113)

### Related issues and pull requests

- [Firefox release PR](https://github.com/mdn/content/pull/31155)
- [BCD PR](https://github.com/mdn/browser-compat-data/pull/21676)

### CODE REVIEW

**Review by estelle — CHANGES_REQUESTED** [review]
Glad you're adding these links.
Just one edit (the same edit, but multiple instances)

**Comment by dletorey** [comment]
> Glad you're adding these links. Just one edit (the same edit, but multiple instances)

@estelle I can not see what this **one edit** is. Perhaps I'm missing something.

File: files/en-us/web/html/global_attributes/popover/index.md [hunk 1]
```diff
@@ -44,3 +44,7 @@ The following renders a button that will open a popover element when activated.
 ## See also
 
 - {{domxref("Popover API", "Popover API", "", "nocode")}}
+- [`popovertarget`](/en-US/docs/Web/HTML/Element/button#popovertarget) HTML `button` attribute
+- [`popovertargetaction`](/en-US/docs/Web/HTML/Element/button#popovertargetaction) HTML `button` attribute
```
Comment (estelle) [inline files/en-us/web/html/global_attributes/popover/index.md:48]:
```suggestion
- [`popovertargetaction`](/en-US/docs/Web/HTML/Element/button#popovertargetaction) HTML attribute
```
It's both for `<button>` and `i<nput type=button>`. All occurences

**Review by estelle — APPROVED** [review]
Thanks! 🎉

**Pull request merged**